vividus-soft-assert/src/main/java/org/vividus/softassert/SoftAssert.java

```diff
@@ -288,6 +289,19 @@ public void verify() throws VerificationError
         }
     }
 
+    @Override
+    public void verify(Pattern pattern) throws VerificationError
+    {
+        if (getAssertionCollection().getAssertionErrors().stream()
```

Comment:
put here NOT op and move verify inside if condition? just to reduce number of code lines by 1

Reply:
I will not, I like the readability of the current version